src/puppetlabs/puppetdb/query/events.clj

```diff
@@ -112,7 +120,11 @@
                                                    (:distinct_start_time query-options)
                                                    (:distinct_end_time query-options))
                                   (default-select select-fields where params))
-        paged-select (jdbc/paged-sql sql paging-options)
+        paged-select (cond-> (jdbc/paged-sql sql paging-options)
```

Comment:
Looking at this code I saw https://github.com/puppetlabs/puppetdb/pull/1453/files#diff-3c835dc023a53db2db8d483f3e6a7da6R118, that looks wrong, shouldn't it be `[sql & params]`?